### ISSUE TITLE
Using keyboard keys not able to invoke "Error" Button.: A11y_Typespec_Playground_Keyboard

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/typespec#8560]
### GIT Hub Tags:
#A11yTCS; #A11ySev1; #TypeSpec; #GH_TypeSpec_Web_Sept2025; #DesktopWeb; #A11yMAS; #WCAG2.1.1; #Keyboard; #AILimited; #Win11; #FTP; #ChromiumEdge; 

### Environment and OS details:
Application name: typespec.io
URL: https://[typespec.io](https://typespec.io/)/
ChromiumEdge Version: 140.0.3485.66 (Official build) (64-bit)

### Reproduction Steps:

1. Open url "https://[typespec.io](https://typespec.io/playground/?sample=API+versioning)/playground/?sample=API+versioning" and login with valid Credentials.
2. "TypeSpec Playground" Page will get open.
3. Edit the Code with some errors.
4. TAB till Error button in bottom of the editor Error Section.
5. Press Enter or Space key to activate Error button.
6. Observe the issue whether Use is able to invoke the "Error" control via keyboard keys or not.

### Actual behavior:
Keyboard users are not able to invoke the "Error" Button using keyboard keys.

### Expected behavior:
User should be able to invoke the "Error" button using Standard keyboard key (Enter or Space).

### User Impact:
Keyboard users will face difficulty while accessing the controls if they are not able invoked by keyboard.

### Attachment:

https://github.com/user-attachments/assets/e9c461ff-1eb4-4ec1-9beb-f881e04432ad

**Comment by ccbarragan** [comment]
I don't have any design suggestions for this one :) looks like the suggested fix is already described in the video. To fix, the error banner in Playground should be made focusable / selectable for screen reader users.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
https://[typespec.io](https://typespec.io/playground/?sample=API+versioning)/playground/?sample=API+versioning
issue is fixed. hence closing the bug. please find added attachment for reference.
#Closed;

https://github.com/user-attachments/assets/97f26147-fcb5-48be-8af2-1c31db02fa5d
